Cubical/Algebra/CommRing/Ideal/Base.agda

```diff
@@ -82,9 +82,9 @@ module CommIdeal (R' : CommRing ℓ) where
 
  -Closed : (I : CommIdeal) (x : R)
          → x ∈ I → (- x) ∈ I
- -Closed I x x∈I = subst (_∈ I) (useSolver x) (·Closed (snd I) (- 1r) x∈I)
-   where useSolver : (x : R) → - 1r · x ≡ - x
-         useSolver = solve R'
+ -Closed I x x∈I = subst (_∈ I) useSolver (·Closed (snd I) (- 1r) x∈I)
```

Comment:
I would have expected that you can just write `solve! R'` in place here, is there a reason why this is not possible?

Reply:
It is a bit mysterious and might be a deficiency of the reflection mechanism. I turned this into issue #1097

Reply:
This is solved now, thanks to @ncfavier :-)